db/migrate/20170905144239_create_usps_confirmation_codes.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUspsConfirmationCodes < ActiveRecord::Migration[5.1]
+  def change
+    create_table :usps_confirmation_codes do |t|
+      t.integer :profile_id, null: false
+      t.string :otp_fingerprint, null: false
+      t.datetime :code_sent_at, null: false, default: ->{ 'CURRENT_TIMESTAMP' }
```

Comment:
`CURRENT_TIMESTAMP` is the Postgres syntax. It ends up becoming `now()` in the schema. We have to use the proc because otherwise Rails tries to treat it like a primitive and sets the default to `'CURRENT_TIMESTAMP'` instead of `CURRENT_TIMESTAMP`.

ref: https://github.com/rails/rails/issues/27077